### ISSUE TITLE
plot3d() error with Gnuplot 6

### ISSUE THREAD

**Issue by rtoy** [rtoy/maxima#2718]
Imported from SourceForge on 2024-07-06 18:15:44
Created by **[dauti](https://sourceforge.net/u/dauti/)** on 2024-05-01 09:16:48
Original: https://sourceforge.net/p/maxima/bugs/4297

---

Gnuplot 6 was released recently. When plotting with Gnuplot 6, e.g. `plot3d()` creates an error message:

~~~
(%i1) plot3d(x+y,[x,1,4],[y,1,4]);
(%o1)                                false
(%i2) 
gnuplot> set pm3d hidden3d 100 border lw 0.5 lt rgb "#000000"
                           ^
         line 0: invalid pm3d option
~~~

Best regards, Wolfgang

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-06 18:15:46
Created by **[robert_dodier](https://sourceforge.net/u/robert_dodier/)** on 2024-05-07 00:47:06
Original: https://sourceforge.net/p/maxima/bugs/4297/#2d46

---

- **labels**: gnuplot, plot --> gnuplot, plotting, plot3d

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-06 18:15:49
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2024-05-11 07:10:17
Original: https://sourceforge.net/p/maxima/bugs/4297/#ed97

---

Resolved with commit f1fa57565263144e4ca7581facf740807c830fbf

**Comment by rtoy** [comment]
Imported from SourceForge on 2024-07-06 18:15:53
Created by **[peterpall](https://sourceforge.net/u/peterpall/)** on 2024-05-11 07:10:37
Original: https://sourceforge.net/p/maxima/bugs/4297/#793e

---

- **status**: open --> closed
